scrapers_next/nj/committees.py

```diff
@@ -0,0 +1,52 @@
+from spatula import JsonPage, URL
+from openstates.models import ScrapeCommittee
+import json
+import re
+
+class CommitteeList(JsonPage):
+    def process_page(self):
+        com_membership = json.loads(self.data[1][0]['CommitteeMembership'])
+        new_com_list = []
+        for com in self.data[0]:
+            name = com['Code_Description']
+            if com['Code_House'] == 'S':
+                chamber = 'upper'
+            elif com['Code_House'] == 'A':
+                chamber = 'lower'
+            else:
+                chamber = 'legislature'
+
+            #TODO: Check classification and parent
+            new_com = ScrapeCommittee(name = name, chamber=chamber)
+
+            try:
+                members = com_membership['Committees'][com['Comm_Status']]
+            except KeyError:
+                members = []
+            for member in members:
+                name = member['FullName']
+                name_regex = re.compile("(.*?), +(.*)")
+                name_match = name_regex.match(name)
+                new_name = " ".join([name_match.group(2), name_match.group(1)])
+                role = member['Position']
+                if role != "":
```

Comment:
Generally, python encourages using `if not role:` for empty strings. Unless we are okay with `None` being allowed past this gate (i.e. `"" != None`).